File: Assignment/modul-3 codes/12.cpp
#include<iostream>
using namespace std;
int main()
{
    int n,i,j;

    cout<<"Enter the row = ";
    cin>>n;

    for(i=0;i<=n;i++)
    {
        for(j=1;j<=i;j++)
        {
            cout<<"* ";
        }
        cout<<endl;
    }

    return 0;
}

File: Assignment/python assignments/12.py
# Practical Example 4: Print this pattern using nested for loop
# *
# * *
# * * * 
# * * * *
# * * * * *

row = int(input("Enter the number = "))

for i in range(0,row) :
    for j in range(0,i+1) :
        print("*",end=" ")
    print()

